app/controllers/search_controller.rb

```diff
@@ -7,11 +7,8 @@ def index
   def results
     begin
       @searchterm = params[:search].nil? ? sanitize_term(params[:q]) : sanitize_term(params[:search])
-      unless params[:search].nil?
-        @search = ManifestationsSearch.new(@searchterm)
-      else
-        @search = ManifestationsSearch.new(query: @searchterm)
-      end
+      @search = params[:search].present? ? SiteWideSearch.new(@searchterm) : SiteWideSearch.new(query: @searchterm)
```

Comment:
I did not found any places where 'search' param is passed here. So perhaps we can remove this code branch?

Reply:
Could be a remnant of some older code, yes.